Updated bitlocker help link

muralikk opened the pull request:
https://docs.microsoft.com/en-us/windows/security/information-protection/bitlocker/bitlocker-basic-deployment#encrypting-volumes-using-the-bitlocker-control-panel

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

muralikk committed:
Updated bitlocker help link

https://docs.microsoft.com/en-us/windows/security/information-protection/bitlocker/bitlocker-basic-deployment#encrypting-volumes-using-the-bitlocker-control-panel
